src/Translator/Collection.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace TCG\Voyager\Translator;
+
+use Illuminate\Support\Collection as IlluminateCollection;
+
+class Collection extends IlluminateCollection
+{
+    //
```

Comment:
@marktopper, what's the purpose of this file? It has no contents, and doesn't override anything from the `IlluminateCollection`